src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1205,7 +1205,7 @@ void ApplicationManagerImpl::OnStreamingConfigured(
       NaviServiceStatusMap::iterator it = navi_service_status_.find(app_id);
       if (navi_service_status_.end() == it) {
         LOG4CXX_WARN(logger_, "Application not found in navi status map");
-        connection_handler().NotifyServiceStartedResult(false, empty);
+        connection_handler().NotifyServiceStartedResult(app_id, false, empty);
```

Comment:
Alright, looking further, it does appear to be treated this way. You can ignore the previous comment